sphinxcontrib/openapi/openapi30.py

```diff
@@ -262,6 +263,27 @@ def _httpresource(endpoint, method, properties, render_examples):
         for line in param.get('description', '').splitlines():
             yield '{indent}{indent}{line}'.format(**locals())
 
+    # print request content
+    if render_request:
+        request_content = properties.get('requestBody', {}).get('content', {})
+        if request_content:
+            schema = request_content['application/json']['schema']
+            req_properties = json.dumps(schema['properties'], indent=2,
+                                        separators=(',', ':'))
+            yield '{indent}**Request body:**'.format(**locals())
+            yield ''
+            yield '{indent}.. sourcecode:: http'.format(**locals())
+            yield ''
+            yield '{indent}{indent}POST /customers HTTP/1.1'.format(**locals())
```

Comment:
Why `Request URI`, `Host` and `Content-Type` are hardcoded?

Reply:
To remove all this stuff, I'll replaced the lines 275-280 with one line using the "json" highlight instead of "http":
yield '{indent}.. sourcecode:: json'.format(**locals())

It works fine locally, so I'm going to commit this change.